include/mkldnn_types.h

```diff
@@ -1005,7 +1007,7 @@ typedef struct {
     /// The kind of eltwise algorithm. Possible values: #mkldnn_eltwise_relu,
     /// #mkldnn_eltwise_tanh, #mkldnn_eltwise_elu, #mkldnn_eltwise_square,
     /// #mkldnn_eltwise_abs, #mkldnn_eltwise_sqrt, #mkldnn_eltwise_linear,
-    /// #mkldnn_eltwise_bounded_relu, #mkldnn_eltwise_soft_relu,
+    /// #mkldnn_eltwise_bounded_relu, #mkldnn_eltwise_soft_relu, #mkldnn_eltwise_swish and
```

Comment:
Please limit line length to 80 columns for all changed lines

Reply:
I have run clang_format on changes , but also adapted style to surrounding code. Hopefully is good enough